src/ansi-c/expr2c.cpp

```diff
@@ -1811,9 +1811,10 @@ std::string expr2ct::convert_constant(
     if(c_enum_type.id()!=ID_c_enum)
       return convert_norep(src, precedence);
 
-    bool is_signed=c_enum_type.subtype().id()==ID_signedbv;
+    const bool is_signed = c_enum_type.subtype().id() == ID_signedbv;
```

Comment:
Nit pick: Unrelated change?

Reply:
Yes, but we appear to do some, and not others!

Reply:
As a reviewer, the first question I usually ask myself is "why was this line changed" and expect to be able to immediately answer that to myself from the commit message. Sometimes there are additional changes _in those lines_ beyond what's in the commit message. If those are of the nature "added in a `const`" or "fixed up whitespace" then those seem like desirable improvements. Everything else raises eyebrows and usually makes me post questions in here ;-)